docs/src/pages/examples/rag/hybrid-vector-search.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: "Example: Hybrid Vector Search | RAG | Mastra Docs"
+description: Example of using metadata filters with PGVector to enhance vector search results in Mastra.
+---
+
+import { GithubLink } from "../../../components/github-link";
+
+# Hybrid Vector Search
+
+When you combine vector similarity search with metadata filters, you can create a hybrid search that is more precise and efficient.
+This approach combines:
+
+- Vector similarity search to find the most relevant documents
+- Metadata filters to refine the search results based on additional criteria
+
+This example demonstrates how to use hybrid vector search with Mastra and PGVector.
+
+## Overview
+
+The system implements filtered vector search using Mastra and PGVector. Here's what it does:
+
+1. Queries existing embeddings in PGVector with metadata filters
+2. Shows how to filter by different metadata fields
+3. Demonstrates combining vector similarity with metadata filtering
+
+> **Note**: To learn how to create and store embeddings in PGVector, see the [Insert Embeddings in PGVector](./insert-embedding-in-pgvector) guide.
+
+## Setup
+
+### Environment Setup
+
+Make sure to set up your environment variables:
+
+```bash filename=".env"
+POSTGRES_CONNECTION_STRING=your_connection_string_here
+```
+
+### Dependencies
+
+Import the necessary dependencies:
+
+```typescript copy showLineNumbers filename="src/index.ts"
+import { embed } from '@mastra/rag';
+import { PgVector } from '@mastra/vector-pg';
+```
+
+## Vector Store Initialization
+
+Initialize PgVector with your connection string:
+
+```typescript copy showLineNumbers{4} filename="src/index.ts"
+const pgVector = new PgVector(process.env.POSTGRES_CONNECTION_STRING!);
+```
+
+## Example Usage
+
+### Filter by Metadata Value
+
+```typescript copy showLineNumbers{55} filename="src/index.ts"
+// Create embedding for the query
+const { embedding } = await embed('[Insert query based on document here]', {
+  provider: 'OPEN_AI',
+  model: 'text-embedding-ada-002',
```

Comment:
This is an old model that gets returned due to knowledge cutoffs. We want to be using `text-embedding-3-small`

Reply:
Updating these now, and will update references to text-embedding-ada-002 to be text-embedding-3-small for other docs as well